packages/composites/crypto-volatility-index/src/sigmaCalculator.ts

```diff
@@ -46,7 +44,7 @@ export class SigmaCalculator {
       const { strikePrice, midPrice, underlyingPrice } = call
       if (strikePrice.gt(underlyingPrice)) {
         if (!midPrice) return
-        const prevMax: Decimal = Decimal.max(...compareCalls.slice(Math.max(0, idx - 10), idx))
+        const prevMax: Decimal = Decimal.max(...[0, ...compareCalls.slice(Math.max(0, idx - 10), idx)])
```

Comment:
Does this fix the `no date found before` thing?

Reply:
It doesn't fix it but it fixed another issue I was hitting.  I can't remember the exact issue but I'll repro it tomorrow and get back to you.

Reply:
So I removed this and ran the code again and got this error

```
[DecimalError] Invalid argument: undefined
```

I think it's because `[...compareCalls.slice(Math.max(0, idx - 10), idx)]` by itself returns an empty array, which causes `Decimal.max` to error out.